docs/team/sagarsureka.md

```diff
@@ -3,44 +3,64 @@ layout: page
 
+* **Enhancements**:
+  * Added a **quantity field** for food items:
+    * What it does: <br>Allows the user to track the quantity of each food item.
+    * Justification: <br>The features improves the application significantly as it helps manage another entity relevant to food items.
+    * Highlights: <br>This enhancement involved consideration of multiple alternatives of implementation. 
+    Keeping to the criteria of making it user friendly, the `unit` of quantity field was made optional and customisable. 
+    Instead of giving some specfic options, the user can have any word as the unit.
+  * Adding a list for **expiring food items**:
+    * What it does: <br>Displays a list panel consisting of food items expiring within 7 days, in ascending order of expiry date.
+    * Justification: <br>Keeping with the motivation of reducing food wastage by food inventory management, the expiring food list allows the user to make informed choices in their meals and avoid having to waste expired food.
+    * Highlights: <br>This enhancement required the introduction of another list into the GUI of the app.
+    In order to keep the GUI clean and not make it too clustered, the expiring food list panel was kept smaller and the food cards in the list had only the minimum details required to properly differentiate the items from each other.
+
+* **Minor Enhancements**:
+  * Contributed in adapting the AB3 code to SimplyKitchen by removing the address field which was no longer needed. [\#70](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/70)
+  * Improved the responses given to the user by making it more readable and informative. [\#79](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/79)  
+  * Improved the product User Guide by fixing bugs reported in the Mock Practical Exam. [\#174](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/174)  
+
+* **Project Management and contribution to team-based tasks**:
+  * Participated in the planning of the project idea (Management of food inventory to reduce food wastage).
+  * Participated in drafting the initial requirements for SimplyKitchen - *Product Scope, Glossary, User Stories, Use Cases and Non-Functional Requirements.* 
+    This can be found in our [Developer Guide requirements appendix](https://ay2021s1-cs2103t-f13-4.github.io/tp/DeveloperGuide.html#appendix-requirements).
+  * Compiled the comments received on the team's User and Developer Guides from peers.
+  * Responsible for proper integration of the various parts of the software to form a whole.
+  * Responsible to ensure that the project code maintains high coding standards.
```

Comment:
Perhaps this could be "Responsible for ensuring"?

Reply:
Thanks!